data/building-list.yaml

```
@@ -98,6 +98,13 @@
   isBuilding: false
   image: tunnel.png
 
+- id: grand-ensemble
+  title: Grand ensemble
+  description: <a href="#grand-ensemble">voir les explications.</a>
+  isBuilding: true
+  image: grand-ensemble.png
+  distinctionBetweenSingleAndMultipleText: Si le découpage de ces ensembles correspond à un usage largement ancré et diffère de la définition du RNB sur l’aspect de la circulation, alors c’est l’usage qui prime. Par exemple, le découpage d’une construction hospitalière par ailes ou secteurs peut produire plusieurs bâtiments, même s’il est possible de circuler entre eux.
```

```suggestion
  distinctionBetweenSingleAndMultipleText: Si le découpage de ces ensembles correspond à un usage largement ancré et diffère de la définition du RNB sur l’aspect de la circulation, alors l’usage prime. Par exemple, le découpage d’une construction hospitalière par ailes ou secteurs peut produire plusieurs bâtiments, même s’il est possible de circuler entre eux.
```

proposition d'allègement